### ISSUE TITLE
[RFC] Does Texture plan on implementing any UIMenu support for UIButton (or UIAction in general)

### ISSUE THREAD

**Issue by DreamingInBinary** [TextureGroup/Texture#1881]
In iOS 14, using [pull down menus](https://developer.apple.com/design/human-interface-guidelines/ios/controls/pull-down-menus/) is a core UX component that largely replaces action sheets. To help facilitate this, `UIAction` replaces a lot of what previously was the target action pattern. 

Specifically, `UIButton` now has a `menu` property that shows one by default if `showsMenuAsPrimaryAction` is to `YES`. In Texture, `ASButtonNode` doesn't wrap a true `UIButton` but instead uses Texture's own version of `ASControlNode` to handle events. This means to use any pull down menu, you need to use a vanilla `ASDisplayView` and have it wrap a true UIKit button.

My question: Does the Texture team deem this important to pursue? If so, awesome! If not, I will look at coming up with a better solution than what I'm using today (wrapping a button).

Thanks!

**Comment by 45641163** [comment]
#1892 Hapus kueri penelusuran, filter, dan jenis saat ini

**Comment by mycroftcanner** [comment]
@appleguy ?

**Comment by onato** [comment]
@DreamingInBinary are you still using the wrapped UIButton method?

**Comment by DreamingInBinary** [comment]
@onato Yes, I'm not sure there is a better route at the moment. It is doable, so I'm not sure implementing this would be really high up on Texture's list.
